src/gatling/simulations/simulations/adoptionSimulation.scala

```diff
@@ -5,49 +5,123 @@ import io.gatling.core.scenario.Simulation
-
-
-  setUp(
-    adoptionSimulation.inject(rampUsers(30) during (60 minutes))
-    .protocols(httpProtocol)
-  )
+  /* TEST TYPE DEFINITION */
+  /* pipeline = nightly pipeline against the AAT environment (see the Jenkins_nightly file) */
+  /* perftest (default) = performance test against the perftest environment */
+  val testType = scala.util.Properties.envOrElse("TEST_TYPE", "perftest")
+
+  //set the environment based on the test type
+  val environment = testType match {
+    case "perftest" => "perftest"
+    case "pipeline" => "aat"
+    case _ => "**INVALID**"
+  }
+
+  /* ******************************** */
+
+  /* ADDITIONAL COMMAND LINE ARGUMENT OPTIONS */
+  val debugMode = System.getProperty("debug", "off") //runs a single user e.g. ./gradle gatlingRun -Ddebug=on (default: off)
+  val env = System.getProperty("env", environment) //manually override the environment aat|perftest e.g. ./gradle gatlingRun -Denv=aat
+  /* ******************************** */
+
+  /* PERFORMANCE TEST CONFIGURATION */
+  val testDurationMins = 60
+  val numberOfPerformanceTestUsers: Double = 30
+  val numberOfPipelineUsers: Double = 30
+
+  //If running in debug mode, disable pauses between steps
+  val pauseOption: PauseType = debugMode match {
+    case "off" => constantPauses
+    case _ => disabledPauses
+  }
+
+      val httpProtocol = http
+        .baseUrl(BaseURL)
+        .doNotTrackHeader("1")
+        .inferHtmlResources()
+        .silentResources
+
+      before {
+        println(s"Test Type: ${testType}")
+        println(s"Test Environment: ${env}")
+        println(s"Debug Mode: ${debugMode}")
+      }
+
+      val adoptionSimulation = scenario("Adoption")
+        .exitBlockOnFail {
+          exec(_.set("env", s"${env}"))
+            .exec(CreateUser.CreateCitizen)
+            .exec(adoptionScenario.adoptionHomepage)
+            .exec(adoptionScenario.adoptionLogin)
+            .exec(adoptionScenario.adoptionApplyingWith)
+            .exec(adoptionScenario.adoptionDateOfMove)
+            .exec(adoptionScenario.adoptionAgency)
+            .exec(adoptionScenario.adoptionYourDetails)
+            .exec(adoptionScenario.adoptionYourContact)
+            .exec(adoptionScenario.adoptionSecondPersonal)
+            .exec(adoptionScenario.adoptionSecondContact)
+            .exec(adoptionScenario.adoptionBirthCertificate)
+            .exec(adoptionScenario.adoptionCertificateDetails)
+            .exec(adoptionScenario.adoptionPlacementOrder)
+            .exec(adoptionScenario.adoptionBirthMother)
+            .exec(adoptionScenario.adoptionBirthFather)
+            .exec(adoptionScenario.adoptionOtherParent)
+            .exec(adoptionScenario.adoptionSiblingDetails)
+            .exec(adoptionScenario.adoptionFamilyCourt)
+            .exec(adoptionScenario.adoptionUploadDocuments)
+            .exec(adoptionScenario.adoptionReview)
+        }
+        .exec(adoptionScenario.adoptionLogOut)
+
```

Comment:
Maybe add your session output here instead of in the scenario, as it's easier to see/remove.